### PR TITLE
[WIP]Support protobuf3.

### LINKED ISSUE #100: Can we support protobuf3?

### PR DESCRIPTION
**Pull request by MrKingHH**
can support both protobuf3 and protobuf2 now.
close #100

### CODE REVIEW

File: brpc-java-examples/brpc-java-core-examples/src/main/proto/echo3.proto [hunk 1]
```diff
@@ -0,0 +1,18 @@
+syntax="proto3";
+
```
Comment (wenweihu86) [inline brpc-java-examples/brpc-java-core-examples/src/main/proto/echo3.proto]: if you use proto3 syntax, the proto files of built-in protocols on brpc-java-core can also modify to proto3.

File: pom.xml [hunk 2]
```diff
@@ -189,7 +189,7 @@
                             <goal>run</goal>
                         </goals>
                         <configuration>
-                            <protocVersion>2.5.0</protocVersion>
+                            <protocVersion>3.9.0</protocVersion>
```
Comment (wenweihu86) [inline pom.xml]: please modify the different version prefix for proto3, such as proto3-2.5.1

**Comment by MrKingHH** [comment]
close this pr temporarily.

**Comment by wenweihu86** [comment]
@MrKingHH master已支持protobuf 3.x

**Pull request closed without merging**